src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html

```diff
@@ -10,6 +10,9 @@
     <link href="css/site.css" rel="stylesheet" />
     <!--#if PWA -->
     <link href="manifest.json" rel="manifest" />
+    <!--iOS does not support specifying icons for PWAs in app manifests so we
```

Comment:
It might be ok to leave this comment out since this ends up in user's applications.

Reply:
Fair enough. I wanted to provide some information to developers consuming the template about the intent of this line but I suppose that can be documented elsewhere...